### PR TITLE
fix 404 and a few link texts in azure aks example

### PR DESCRIPTION
**Pull request by susanev**

### CODE REVIEW

File: classic-azure-ts-aks-mean/README.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ npm install
 
 One of the interesting aspects of this example is the way it demonstrates how easy it is to use
 Azure resources to configure Kubernetes resources, without the need for intermediate APIs such as
-[OSBA](https://osba.sh/). In particular, this example uses the connection strings exposed by the
+[OSBA](https://github.com/Azure/open-service-broker-azure). In particular, this example uses the connection strings exposed by the
```
Comment (susanev) [inline classic-azure-ts-aks-mean/README.md:97]: is this the best link to use?

**Review by mikhailshilkov — APPROVED** [review]
LGTM. I don't know what OSBA is, but the link looks good.

**Review by jkodroff — APPROVED** [review]

**Comment by ericrudder** [comment]
Open Service Broker (for Azure)

File: classic-azure-ts-aks-mean/README.md (hunk 1 above)
Comment (susanev) [inline classic-azure-ts-aks-mean/README.md:97]:
```suggestion
[Open Service Broker for Azure](https://github.com/Azure/open-service-broker-azure). In particular, this example uses the connection strings exposed by the
```

**Pull request merged**